docs/CSBR.md

```diff
@@ -91,6 +93,10 @@ If a Delegated Third Party fulfills any of the CA's obligations under [Section 4
 
 ### 1.3.5 Other participants
 
+Signing Services MAY support generation of Subscriber Key Pair and maintain security of the Subscriber Private Key.

```

Comment:
Should we leave this just to avoid people making unwarranted assumptions that removing it means it's prohibited?